DQM/Integration/python/clients/ctpps_dqm_sourceclient-live_cfg.py

```diff
@@ -12,15 +12,14 @@
   # for testing in lxplus
   process.load("DQM.Integration.config.fileinputsource_cfi")
   process.source.fileNames = cms.untracked.vstring(
-    'file:/afs/cern.ch/user/j/jkaspar/public/run273062_ls0001-2_stream.root',
+    #'root://eostotem.cern.ch//eos/totem/user/j/jkaspar/04C8034A-9626-E611-9B6E-02163E011F93.root'
```

Comment:
Hi Dmitrijus,

the lines you talk about would not be executed since the `test` variable is set to False:
  https://github.com/CTPPS/cmssw/blob/07882d194c34f8cf39ea91471c3f9ca176af35bc/DQM/Integration/python/clients/ctpps_dqm_sourceclient-live_cfg.py#L5
Nevertheless, if you prefer, I can remove the "test" block completely - let me know.